tox.ini

```diff
@@ -67,7 +67,9 @@ uv_resolution =
   min: lowest-direct
 
 commands =
-  pytest --durations=20 --durations-min=1.0 --cov-report term-missing {posargs:-n auto}
+  pytest --durations=20 --durations-min=1.0 --cov-report term-missing \
+  min,pre: -Wignore \
```

Comment:
This warning setting takes precedence over those in `tool.pytest.ini_options`. For `min`, we expect a lot of warnings that the older versions of the libraries we're testing of course cannot resolve. For `pre`, we expect many `PendingDeprecationWarning`s and `DeprecationWarning`s before our upstreams even have time to address.